### DISCUSSION TITLE
requesting partially available

### DISCUSSION THREAD

**Discussion by nmbrg** [sct/overseerr#3575] [Help]
not sure if this is a bug or a feature request, or if im just dumb.

if i request a series (lets say tom clancy's jack ryan) and later deletes season 1-3 and only keeps the newest season (4)
then season 4 is still marked as available in overseerr and season 1-3 is marked as "requested" for some reason.

so, now no one can request the older seasons, if they want to watch them.

is this by design or some bug? 

wouldnt it make more sense to mark it as unavailable (since its in fact is unavailable/does not exist anymore) so people can request it again?

and this is not only for this specific series, i just took that one as an example. this happens on all series since i remove old stuff to clear out storage for new stuff. and frankly how often do you look at old stuff...

**Comment by supermarkert** [comment]
I had a similar issue, and have been playing around with it for a while, and think I found a workaround:

1. In Sonarr, for the (deleted) season in question, click the flag to Un-monitor the season. 
2. In Overseerr, go to the series in question. Choose the gear icon on the right side, and choose "Clear Data".
3. In Overseerr Settings --> Jobs & Cache, do a Sonarr scan.

You should now be able to request older seasons. In my Sonarr settings (as I assume most people do), when a particular episode is deleted from disk, that episode automatically becomes Unmonitored. However, it doesn't Unmonitor the entire season.
Overseerr seems to see this Monitor checkmark for the season and determine that "Hey, it's already requested, don't request again".  
I'm not sure of any way yet to do this automatically, because it's just the way Sonarr works. If there was a setting in Sonarr to "Unmonitor the season once all episodes are unmonitored", then I think the problem would go away.

Edit: Actually, the problem wouldn't go away completely, because Overseerr still thinks the request is open. So you'd still have to manually "Clear Data" in order to start over.
